### DISCUSSION TITLE
Get stars for your repo

### DISCUSSION THREAD

**Discussion by pu-raihan** [drknzz/GitHub-Achievements#187] [Q&A]
Hey guys,
This is my project- Social media website using React [CamBuzz](https://github.com/pu-raihan/CamBuzz)
Take a look and **Star** this repo. I'll be doing the same for you thanks

**Comment by maciekpastuszak** [comment]
Hey, very cool project!

**Reply by pu-raihan** [reply]
Thanks buddy, appreciate it

**Comment by HarshDekate** [comment]
give  star to my repos i will give you badges

**Reply by pu-raihan** [reply]
Sure buddy

**Comment by HarshDekate** [comment]
give  star to my repos i will give you badges

**Reply by pu-raihan** [reply]
Done

**Reply by HarshDekate** [reply]
thanks
